### PR TITLE
Microsoft.XNARedist version 4.0.30901.0

### PR DESCRIPTION
**Pull request by Masamune3210**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/27364)

### CODE REVIEW

File: manifests/m/Microsoft/XNARedist/4.0.30901.0/Microsoft.XNARedist.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.XNARedist
+PackageVersion: 4.0.30901.0
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+PackageName: Microsoft XNA Framework Redistributable 4.0 Refresh
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/XNARedist/4.0.30901.0/Microsoft.XNARedist.locale.en-US.yaml:8]:
```suggestion
PackageName: Microsoft XNA Framework Redistributable Refresh
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/m/Microsoft/XNARedist/4.0.30901.0/Microsoft.XNARedist.locale.en-US.yaml (hunk 1 above)
Comment (Masamune3210) [reply manifests/m/Microsoft/XNARedist/4.0.30901.0/Microsoft.XNARedist.locale.en-US.yaml:8]: Thought PackageName needed to match what was shown in the Add and Remove Programs list

Comment (Masamune3210) [reply manifests/m/Microsoft/XNARedist/4.0.30901.0/Microsoft.XNARedist.locale.en-US.yaml:8]: image

Comment (ItzLevvie) [reply manifests/m/Microsoft/XNARedist/4.0.30901.0/Microsoft.XNARedist.locale.en-US.yaml:8]: without the version number included for now. It'll be fixed up later in 1.1.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/27364#discussion_r706271074

File: manifests/m/Microsoft/XNARedist/4.0.30901.0/Microsoft.XNARedist.locale.en-US.yaml (hunk 1 above)
Comment (Masamune3210) [reply manifests/m/Microsoft/XNARedist/4.0.30901.0/Microsoft.XNARedist.locale.en-US.yaml:8]: aah ok got it

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**